topology/probes/k8s/networkpolicy.go

```diff
@@ -254,7 +345,7 @@ func (n *networkPolicyProbe) onNodeUpdated(objNode *graph.Node) {
 			logging.GetLogger().Debugf("can't find %s", dumpNetworkPolicy(np))
 			continue
 		}
-		syncLink(n.graph, npNode, objNode, n.isSelected(np, obj))
+		n.updateLinks(npNode, np)
```

Comment:
It seems that when there for every creation/update of a pod or a namespace, the links for every network policy is computed. The previous version tried to only compute the links for the affected namespace or pod

Reply:
I have passed the graph.Node object (namespace or pod) and now use it so as to focus only on the links connecting this object to all networkpolicy objects.